foxglove/console/client.go

```diff
@@ -195,6 +196,31 @@ func (c *FoxgloveClient) post(
 	return nil
 }
 
+func (c *FoxgloveClient) delete(endpoint string) error {
+	req, err := http.NewRequest(http.MethodDelete, c.baseurl+endpoint, nil)
+	if err != nil {
+		return fmt.Errorf("failed to create request: %w", err)
+	}
+
+	resp, err := c.authed.Do(req)
+	if err != nil {
+		return fmt.Errorf("request failed: %w", err)
+	}
```

Comment:
this and similar methods should be calling `defer resp.Body.Close()` if there's no error. We're missing that in a few cases.

Reply:
Added here and in `post`